pkg/resources/server.go

```diff
@@ -358,3 +368,45 @@ func (rs *resourceServer) NotifyRegistrationStatus(ctx context.Context, regstat
 	}
 	return &registerapi.RegistrationStatusResponse{}, nil
 }
+
+func (rs *resourceServer) UpdateDevices(devices []types.PciNetDevice) {
+	// Get device spec
+	deviceSpec := utils.GetDevicesSpec(devices)
+
+	// If not devices not changed skip
+	if !utils.DevicesChanged(rs.deviceSpec, deviceSpec) {
```

Comment:
you can have `DevicesChanged` as non public method under resourceServer , or just in this file

Reply:
changed to private function in same file